services/slam/slam.go

```diff
@@ -101,10 +101,10 @@ func HelperConcatenateChunksToFull(f func() ([]byte, error)) ([]byte, error) {
 }
 
 // PointCloudMapFull concatenates the streaming responses from PointCloudMap into a full point cloud.
-func PointCloudMapFull(ctx context.Context, slamSvc Service) ([]byte, error) {
+func PointCloudMapFull(ctx context.Context, slamSvc Service, returnEditedMap bool) ([]byte, error) {
 	ctx, span := trace.StartSpan(ctx, "slam::PointCloudMapFull")
 	defer span.End()
-	callback, err := slamSvc.PointCloudMap(ctx)
+	callback, err := slamSvc.PointCloudMap(ctx, returnEditedMap)
```

Comment:
I think this should be a default false, if a user added new bounds outside the map in the edit, I don't think I want motion to know about that. cc @raybjork

Reply:
It’s just a helper function for any users doing SLAM in go. We also use this in cloudslam when generating the package at the end of a run.